scripts/cmd_args.py

```diff
@@ -236,6 +236,17 @@ def add_inference_options(parser):
     add_common_options(parser)
     parser.add('--config', required=False,
                is_config_file=True, help='config file path')
+    parser.add('--regions', required=True, type=type_or_none_fn(str),
```

Comment:
Should we specify that the length of the region needs to be a multiple of `--interval_size`?

Reply:
good point. Done